lib/ssh_scan/banner.rb

```diff
@@ -29,6 +29,8 @@ def ssh_lib_guess()
         return SSHScan::SSHLib::CiscoSSH.new()
       when /ROS/i
         return SSHScan::SSHLib::ROSSSH.new()
+      when /DOPRASSH/i
```

Comment:
@claudijd Do you happen to know a fingerprint that includes "DOPRASSH" literally? Just curious.